backend/core/migrations/0011_user_votingright_isnow_trustscore.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.0.8 on 2022-10-20 14:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0010_alter_user_voting_right"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Won't the data be lost if we remove and recreate in the same migration?

Reply:
Yes, but we currently have no data in production. If I remember correctly I can rename the field instead of deleting it. I'll do it.